server/controllers/resetController.js

```diff
@@ -0,0 +1,75 @@
+import { config } from 'dotenv';
+import authHelper from '../helpers/authHelpers';
+
+config();
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * password reset and update.
+ */
+class ResetController {
+  /**
+     * This method find the user email request.
+     * @param {object} req The user's request.
+     * @param {object} res The response.
+     * @returns {object} The status and some data of the user.
+     */
+  static async sendResetPasswordLink(req, res) {
+    const { email } = req.body;
+    const registeredUser = await authHelper.findUser(email);
+    const task = 'password-reset';
+
+
+    if (registeredUser) {
+      const token = await authHelper.createToken({
+        id: registeredUser.id,
+        email
+      });
+
+      await authHelper.sendMail(email, token, task);
+      res.status(200).send({
+        status: 200,
+        message: 'check your email to reset password!'
```

Comment:
Consider hanging the message to '**Password reset link has been sent to your email. Check it out!**'

Reply:
updated